.travis.yml

```diff
@@ -1,3 +1,6 @@
 
 script:
+  - make venv
+  - source .venv/bin/activate
```

Comment:
I think you wouldn't need to do it in a virtualenv inside Travis. It's already an isolated environment. So, just install whatever you need here and it should be fine.

Reply:
Okay.